internal/function/uast_utils.go

```diff
@@ -192,6 +214,10 @@ func nodesFromBlobArray(data interface{}) ([]*uast.Node, error) {
 	}
 
 	arr := data.([]interface{})
+	if len(arr) == 0 {
```

Comment:
this check is needed?

Reply:
Tests for the old serialization checked that uast and uast_mode returned nil in some cases. Without these checks those cases return a slice of length 0 for the new serialization.